x-pack/test/api_integration/apis/ml/filters/update_filters.ts

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { USER } from '../../../../functional/services/ml/security_common';
+import { COMMON_REQUEST_HEADERS } from '../../../../functional/services/ml/common';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext) => {
+  const supertest = getService('supertestWithoutAuth');
+  const ml = getService('ml');
+
+  const items = ['104.236.210.185'];
+  const validFilters = [
+    {
+      filterId: 'filter_power',
+      requestBody: { description: 'Test delete filter #1', items },
+    },
+    {
+      filterId: 'filter_viewer',
+      requestBody: { description: 'Test delete filter (viewer)', items },
+    },
+    {
+      filterId: 'filter_unauthorized',
+      requestBody: { description: 'Test delete filter (unauthorized)', items },
+    },
+  ];
+
+  describe('update_filters', function () {
+    const updateFilterRequest = {
```

Comment:
I'd suggest to name this `updateFilterRequestBody`.

Reply:
Fixed in https://github.com/elastic/kibana/pull/72564/commits/121d1e5fdd860b91516a4ae3e2023cec45205c9b